src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -477,6 +478,58 @@ def get_config_actions(self):
             ),
         ]
 
+    def update_registration_with_confirmation_email(
+        self, submission: Submission, options: RegistrationOptions
+    ) -> dict:
+        logger.info("stuf_zds_finalise_registration_started", options=options)
```

Comment:
Probably best to update the event here?

```suggestion
        logger.info("update_registration_with_confirmation_email_started", options=options)
```
the structlog context should already have bound the plugin in the outer/generic code, so I think this is sufficient to identify which plugin ran.

Reply:
Forgot to change these when renaming